background.js

```diff
@@ -1,5 +1,21 @@
 const sigarraRegex = /.*:\/\/sigarra\.up\.pt\/feup\/.*/;
 
+// Add default values for each option here
+const popupOptions = [
+  {
+    key1: "on"
+  },
+  {
+    shortcuts: "on"
+  },
+  {
+    autoLogin: "off"
+  },
+  {
+    font: "on"
+  },
+]
```

Comment:
What do you think about this beeing a object? And then looping thought its keys and values?

Reply:
That shouldn't be a problem, I'd suggest even creating a separate JSON file so that part of the setup is even more separate from the logic. What do you think?

Reply:
Seems a perfect solution!